downstream/modules/aap-on-azure/proc-azure-resource-quotas.adoc

```diff
@@ -0,0 +1,61 @@
+////
+Base the file name and the ID on the module title. For example:
+* file name: con-my-concept-module-a.adoc
+* ID: [id="con-my-concept-module-a_{context}"]
+* Title: = My concept module A
+////
+
+[id="proc-azure-resource-quotas"]
+
+= Azure resource quotas and infrastructure limits
+
+Microsoft imposes resource limits within each Azure region. The limit for CPU is the most likely to impact Red Hat Ansible Automation Platform on Microsoft Azure.
+
+Before you install Ansible Automation Platform for Azure, ensure that you have capacity to deploy the managed application into your desired region.
+Refer to xref:con-aap-azure-infrastructure-usage[Azure infrastructure usage] for infrastructure requirements.
+
+== Regional vCPU limits
+
+The Azure resources used during the deployment of the managed application temporarily exceed the resource requirements in xref:con-aap-azure-infrastructure-usage[Azure infrastructure usage]. The `Total Regional vCPUs` quota is temporarily consumed when deploying the managed application.
+
+Every Azure region has a separate `Total Regional vCPUs` quota. To prevent installation failure, ensure that you have at least 8 available vCPUs in the Azure region where you want to deploy the managed application.
+
+The following steps describe how to view the resource quotas for your subscription the Azure console:
+
+. In the Azure console, search for *Quotas* and open the *My Quotas* page.
+. Select the region where you wish to deploy the managed application to view your allocation and usage metrics for that region.
++
+Ensure that you have selected a single region. Viewing all regions at once will not show the limitations of a single Azure region.
```

Comment:
Is this meant to be a nested item in the list? If so:

.. Ensure that you have...